src/App/components/Section/components/CookiePolicy/CookiePolicy.tsx

```diff
@@ -0,0 +1,51 @@
+import style from './CookiePolicy.module.scss';
+import useTheme from '../../../../hooks/useTheme';
+import { Button, Modal } from '@mui/material';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+const CookiePolicy = () => {
+  const bodyTheme = useTheme(style.lightBody, style.darkBody);
+  const [isModalOpen, setOpen] = useState(true);
+  const { t } = useTranslation();
+
+  const handleClick = () => {
+    localStorage.setItem('cookies', 'true');
+    setOpen(false);
+  };
+
+  useEffect(() => {
+    if (localStorage.getItem('cookies') === null) {
+      localStorage.setItem('cookies', 'false');
+    }
+    if (localStorage.getItem('cookies') === 'false') {
+      setOpen(true);
+    }
+    if (localStorage.getItem('cookies') === 'true') {
+      setOpen(false);
+    }
```

Comment:
I think, we can simplify here, like this:
```
const cookies = localStorage.getItem('cookies');

if(!cookies) {
    localStorage.setItem('cookies', false);
} else {
    setOpen(!JSON.parse(cookies));
}
```

Reply:
This is now complete